src/sentry/api/serializers/models/rule.py

```diff
@@ -39,6 +40,37 @@ def _is_filter(data):
     return rule_cls.rule_type == "filter/event"
 
 
+class RuleCreatedBy(TypedDict):
+    id: int
+    name: str
+    email: str
+
+
+class RuleSerializerResponse(TypedDict, total=False):
```

Comment:
Should all fields in this TypedDict be optionally not set with the `total=False`? It seems a bit odd that the API response could have nothing in it

Reply:
I was under the impression that this was necessary to return a response without the optional fields, didn't realize it meant you could omit fields that weren't optional! I'll remove it, but I guess that means I'll need to include the optional fields in the examples (just with `None` as the value).